WooCommerce/Classes/ViewRelated/Authentication/AccountCreationForm.swift

```diff
@@ -92,7 +92,7 @@ struct AccountCreationForm: View {
                     .renderedIf(viewModel.currentField == .password)
 
                     // Terms of Service link.
-                    AttributedText(tosAttributedText, enablesLinkUnderline: true)
```

Comment:
I'm not sure if we should remove the `enableLinkUnderline` completely now or not, WDYT?

Reply:
I am OK with keeping it inside `AttributedText` as a feature. IMO, cleaning up `AttributedText` sounds like it is out of scope for this project.